### DISCUSSION TITLE
Is it possible to get (personalized) news from users' followed sites?

### DISCUSSION THREAD

**Discussion by phoob** [microsoft-search/pnp-modern-search#1435] [Q&A] [answered]
I know it is possible to use Microsoft's news web part to display "recommended [news] for the current user", but can't see any option to only show news from a user's followed sites. 

Since I get complaints from users seeing news that they are not interested in in the news feed on our portal, I wonder if it would be possible to list news from a user's followed sites using PnP Modern Search?

**Comment by wobba** [comment, marked as answer]
Not today. We would need to implement a call to fetch the users followed sites and then add that as a filter to the query.

**Reply by phoob** [reply]
Ok thanks. That would be a useful filter :)

**Reply by wobba** [reply]
Question, are your users using "Follow" a lot for sites?

**Reply by phoob** [reply]
> Question, are your users using "Follow" a lot for sites?

We encourage the users to follow sites, since (we believe) that influence which news are recommended for them, but I don't have any stats on eg. the average number of sites followed per user.
